azurerm/internal/services/compute/ssh_keys.go

```diff
@@ -24,7 +25,7 @@ func SSHKeysSchema(isVirtualMachine bool) *schema.Schema {
 					Type:         schema.TypeString,
 					Required:     true,
 					ForceNew:     isVirtualMachine,
-					ValidateFunc: validation.StringIsNotEmpty,
+					ValidateFunc: ValidateSSHKey,
```

Comment:
this is used in the new VM resources not the old one (AFAIK) - could we update the documentation for those in addition/instead?

Reply:
Docs for the new resources already make mention of the restriction for key signature and length I believe, but I'll double check.